### ISSUE TITLE
Venus vault endpoint is not secured

### ISSUE THREAD

**Issue by SolidSnackDrive** [SENG426-Team4/vega-web#44]
# Description
Anyone is able to send requests to this endpoint, regardless of role.
# Origin of bug

# Steps to reproduce
1. Use postman to send a post request to this endpoint
# Story Points:
SP: 3
# Priority:
High

**Comment by SolidSnackDrive** [comment]
Probably makes sense to link this to a PR, not sure which one though @mschaldias

**Comment by mschaldias** [comment]
@SolidSnackDrive yeah not sure if it should be Feature/vault crud #8 or a new one for next milestone. But I couldn't figure out how to link issues here to PRs in vega-spring
